.gitignore

```diff
@@ -12,3 +12,4 @@
 /tags
 /tmp/*
 .thrc
+/vendor/
```

Comment:
removing bower is not the same as removing everything from the vendor, replace this with just the bower folder.